src/scripts/components/Tooltip.jsx

```diff
@@ -0,0 +1,210 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactDOMServer from 'react-dom/server';
+import { TimelineMax } from 'gsap';
+
+import { UtilitySystem } from '../UtilitySystem';
+
+class Tooltip extends React.Component {
+  // @NOTE Attaching event listeners here is not ideal, but `onMouseEnter` (and `onMouseOver`) is not reliably fired; as well as the context of the currentTarget being incorrect.
+  componentDidMount() {
+    const tooltipTrigger = this.getTooltipTrigger();
+
+    tooltipTrigger.addEventListener('mouseenter', (e) => {
```

Comment:
Don't forget to remove the event listener on `componentWillUnmount` for `mouseenter` and `mouseleave`.

Reply:
👍 Thanks for the heads up, added that in.